test/Scripts.Integration.Test/integration-add-sentry.ps1

```diff
@@ -0,0 +1,49 @@
+﻿param($path)
+
+. ./test/Scripts.Integration.Test/IntegrationGlobals.ps1
+
+ShowIntroAndValidateRequiredPaths "True" "Add Sentry" $path
+
+Write-Output "Removing Log"
+ClearUnityLog
+
+Write-Host -NoNewline "Injecting Editor script"
+$stdout = New-Item -Path "$NewProjectAssetsPath" -Name "Editor" -ItemType "directory" -ErrorAction Stop
+Copy-Item "$IntegrationScriptsPath/SentrySetup.cs"      -Destination "$NewProjectAssetsPath/Editor" -ErrorAction Stop
+Copy-Item "$IntegrationScriptsPath/SentrySetup.cs.meta" -Destination "$NewProjectAssetsPath/Editor" -ErrorAction Stop
+Write-Output " OK"
+
+Write-Host -NoNewline "Applying Sentry package to the project:"
+$UnityProcess = Start-Process -FilePath "$Global:UnityPath/$Unity" -ArgumentList "-batchmode", "-projectPath ", "$NewProjectPath", "-logfile", "$NewProjectLogPath/$LogFile" -PassThru
```

Comment:
Instead of copying these `.cs` files and then simply launching Unity, you open unity and tell it to run a method.
We do that in the repo with `-executeMethod`, see:

https://github.com/getsentry/sentry-unity/blob/main/samples/unity-of-bugs/Assets/Editor/Builder.cs#L55

https://github.com/getsentry/sentry-unity/blob/d6883adb504363390492961108fc3340c9783659/Directory.Build.targets#L169